[DEM] clang compiler warnings from nightly build

~~~
/home/ubuntu/Kratos/applications/DEMApplication/custom_utilities/../custom_elements/spheric_continuum_particle.h:111:22: warning: 'Kratos::SphericContinuumParticle::CalculateOnContactElements' hides overloaded virtual function [-Woverloaded-virtual]
/home/ubuntu/Kratos/applications/DEMApplication/custom_utilities/analytic_tools/analytic_face_watcher.h:200:16: warning: private field 'mRelVelNormalxMass' is not used [-Wunused-private-field]
/home/ubuntu/Kratos/applications/DEMApplication/custom_utilities/analytic_tools/analytic_face_watcher.h:201:16: warning: private field 'mRelVelTangentialxMass' is not used [-Wunused-private-field]
/home/ubuntu/Kratos/applications/DEMApplication/custom_elements/spheric_continuum_particle.h:111:22: warning: 'Kratos::SphericContinuumParticle::CalculateOnContactElements' hides overloaded virtual function [-Woverloaded-virtual]
~~~

--- a/applications/DEMApplication/custom_elements/spheric_continuum_particle.cpp
+++ b/applications/DEMApplication/custom_elements/spheric_continuum_particle.cpp
@@ -403,7 +403,7 @@ namespace Kratos {
                 total_local_elastic_contact_force[0] = LocalElasticContactForce[0] + LocalElasticExtraContactForce[0];
                 total_local_elastic_contact_force[1] = LocalElasticContactForce[1] + LocalElasticExtraContactForce[1];
                 total_local_elastic_contact_force[2] = LocalElasticContactForce[2] + LocalElasticExtraContactForce[2];
-                CalculateOnContactElements(i, total_local_elastic_contact_force, contact_sigma, contact_tau, failure_criterion_state, acumulated_damage, time_steps);
+                CalculateOnContinuumContactElements(i, total_local_elastic_contact_force, contact_sigma, contact_tau, failure_criterion_state, acumulated_damage, time_steps);
             }
 
             if (this->Is(DEMFlags::HAS_STRESS_TENSOR) /*&& (i < mContinuumInitialNeighborsSize)*/) {
@@ -823,7 +823,7 @@ namespace Kratos {
         else return 0.0;
     }
 
-    void SphericContinuumParticle::CalculateOnContactElements(size_t i, double LocalElasticContactForce[3], double contact_sigma, double contact_tau, double failure_criterion_state, double acumulated_damage, int time_steps) {
+    void SphericContinuumParticle::CalculateOnContinuumContactElements(size_t i, double LocalElasticContactForce[3], double contact_sigma, double contact_tau, double failure_criterion_state, double acumulated_damage, int time_steps) {
 
         KRATOS_TRY
 

--- a/applications/DEMApplication/custom_elements/spheric_continuum_particle.h
+++ b/applications/DEMApplication/custom_elements/spheric_continuum_particle.h
@@ -108,7 +108,7 @@ namespace Kratos
         virtual double CalculateMaxSearchDistance(const bool has_mpi, const ProcessInfo& r_process_info);
         virtual bool OverlappedParticleRemoval();
         virtual void CalculateMeanContactArea(const bool has_mpi, const ProcessInfo& r_process_info);
-        virtual void CalculateOnContactElements(size_t i_neighbour_count, double LocalElasticContactForce[3],
+        virtual void CalculateOnContinuumContactElements(size_t i_neighbour_count, double LocalElasticContactForce[3],
                                                 double contact_sigma, double contact_tau, double failure_criterion_state, double acumulated_damage, int time_steps);
 
 

--- a/applications/DEMApplication/custom_utilities/analytic_tools/analytic_face_watcher.h
+++ b/applications/DEMApplication/custom_utilities/analytic_tools/analytic_face_watcher.h
@@ -197,8 +197,6 @@ class FaceHistoryDatabase // It holds the historical information gathered for a 
         std::vector<int> mId2;
         std::vector<double> mRelVelNormal;
         std::vector<double> mRelVelTangential;
-        double mRelVelNormalxMass;
-        double mRelVelTangentialxMass;
 };
 
 void ClearData();

